.github/workflows/build_slim_packages.yaml

```diff
@@ -29,17 +29,14 @@ jobs:
       fail-fast: false
       matrix:
         profile:
-        - emqx
-        - emqx-enterprise
+        - ["emqx", "el7"]
+        - ["emqx-enterprise", "ubuntu20.04"]
```

Comment:
@zmstone missed your comment in https://github.com/emqx/emqx/pull/9497
do we need to swap these? I.e. build emqx-enteprise-el7 and emqx-ubuntu-20.04 instead?

Reply:
It’s ok